onnxruntime/contrib_ops/cpu/tokenizer.cc

```diff
@@ -129,7 +129,7 @@ Status Tokenizer::CharTokenize(OpKernelContext* ctx, size_t N, size_t C,
     if (!utf8_validate(reinterpret_cast<const unsigned char*>(s.data()), s.size(),
                        tokens)) {
       return Status(common::ONNXRUNTIME, common::INVALID_ARGUMENT,
-                    "Input string contains invalid utf8 chars: " + s);
+                    "Input string contains invalid utf8 chars");
```

Comment:
If we need to be careful about not including the input string in the status here you should add comments to that effect to prevent it being added back by someone trying to make the error more helpful.

Reply:
Done